ExpenseTracker/ExpenseTracker/Views/Shared/_Layout.cshtml

```diff
@@ -38,6 +38,7 @@
                     </a>
                 </div>
                 <div>
+                    <div style="display: flex; align-items: center;">
```

Comment:
maybe `className="d-flex align-items-center"`

Reply:
No , he is doing well